blocks/filter-group/filter-group.css

```diff
@@ -29,9 +29,10 @@ effect on filter buttons to indicate horizontal scroll */
   display: none;
 }
 
-/* Adds left padding on focus to avoid outline cutoff*/
-.filter-group__list:has(.filter-group__button:first-child:focus-visible) {
-  padding-inline-start: 0.5rem;
+/* Adds left padding on focus to avoid outline cutoff on the first element. */
+.filter-group__list:has(.filter-group__button:focus-visible) {
+  /* Max focus outline size plus its outline offset. */
+  padding-inline-start: 4px;
```

Comment:
I'd love to do `calc(var(--focus-outline-size) + var(--focus-outline-offset))`, but those are currently scoped to specific elements in `styles.css`. Could we move those custom properties to `:root` to allow this, or just leave this as-is?

Reply:
uh yeah absolutely move those up, we might need to rename them if they seem too vague